### DISCUSSION TITLE
Using python sqlachemy/alembic

### DISCUSSION THREAD

**Discussion by theFong** [planetscale/discussion#60] [Q&A] [answered]
Hello,

I'm having trouble connecting python sqlachemy/alembic to a database branch. Are there any examples of this?

I have a database `sandbox` and am using the connection string `mysql+pymysql://root:@127.0.0.1:3306/sandbox` and am getting the error message `"unknown database 'sandbox'"`

**Comment by Anders-PlanetScale** [comment, marked as answer]
Howdy @theFong when you created your DB did you check the box to "Include tutorial data set" if so the default DB gets renamed from your PSDB name to onboarding-demo. 

You should be able to confirm this by using `pscale shell` or connecting via mysql client and running `show databases;`

I would recommend creating a new PSDB without the tutorial data set and seeing if you continue to have issues.
